### ISSUE TITLE
Split off php-variables.data to PL3 rule

### ISSUE THREAD

**Issue by fzipi** [crstest01/owasp-modsecurity-crs#362]
_Issue originally created by user csanders-git on date 2016-06-15 19:48:08.
            Link to original issue: https://github.com/SpiderLabs/owasp-modsecurity-crs/issues/357._

The use of the ARG_NAMES value within 933130 causes extensive issues with Wordpress due to their parameter names. The suggestion is the make a version with ARGS_NAMES at paranoid level 2.
Is the use of such words common within attacks using register_globals? **at user lifeforms** 
![screenshot from 2016-06-15 15-39-10](https://cloud.githubusercontent.com/assets/4632287/16094678/5556192a-330f-11e6-8484-b1b8f0722438.png)

**Comment by fzipi** [comment]
_User lifeforms commented on date 2016-06-15 20:02:04:_

**at user csanders-git** Good find. To be honest there's a lot in the `php-variables.data` file that I don't overly care about. About the top half of the data file is interesting from an attacker standpoint (though easy to evade, cause if you can evaluate these strings, you probably already have RCE and you can use variable variables or other tricks). Starting at `AUTH_TYPE` the data file gets kinda meh. The `PHP_` strings would hopefully not turn up so often in FP though, so they could be nice to keep for now.

We could split off the losers to a higher paranoia level if you like to keep them. Maybe PL3 since they kinda attract FP and are not overly important/typical for attack code? Just by educated guess, I think they'll be a lot worse than many other PL2 rules in terms of FP. But if you guys think it's worth it for PL2, that's fine with me.

To recap I'd propose moving these terms to PL3.

```
AUTH_TYPE
HTTP_ACCEPT
HTTP_ACCEPT_CHARSET
HTTP_ACCEPT_ENCODING
HTTP_ACCEPT_LANGUAGE
HTTP_CONNECTION
HTTP_HOST
HTTP_KEEP_ALIVE
HTTP_REFERER
HTTP_USER_AGENT
HTTP_X_FORWARDED_FOR
ORIG_PATH_INFO
PATH_INFO
PATH_TRANSLATED
QUERY_STRING
REQUEST_URI
```

**Comment by fzipi** [comment]
_User csanders-git commented on date 2016-06-15 20:14:16:_

I'm inclined to agree with you but perhaps these make more sense while trying to detect code. As you said the FP rate should be low (within parameters). I think it is just the locations that will lead to increased false positives.

**Comment by fzipi** [comment]
_User lifeforms commented on date 2016-06-15 20:18:48:_

**at user csanders-git** That's true, but stuff also ends up in `ARGS_NAMES` when it's just passed as a query string (e.g. `/foo?PAYLOAD_HERE`), which is effective in a lot of vulnerabilities, so I think checking that variable is pretty much a must for most rules. There's a lot of options, but I would rather start with moving around the less important terms and see how we fare then.

**Comment by fzipi** [comment]
_User csanders-git commented on date 2016-06-15 20:39:17:_

That sounds pretty reasonable - I can make a PR with the modified files ... is that pk with you?

**Comment by fzipi** [comment]
_User lifeforms commented on date 2016-06-15 20:41:40:_

**at user csanders-git** Sure!

**Comment by fzipi** [comment]
_User dune73 commented on date 2016-06-16 11:48:35:_

ACK

**Comment by fzipi** [comment]
_User lifeforms commented on date 2016-08-07 08:31:03:_

Closing this, PR is in #473
